src/test/java/ua/bookstore/online/utils/ConstantAndMethod.java

```diff
@@ -13,40 +15,107 @@
 import ua.bookstore.online.dto.book.CreateBookRequestDto;
 import ua.bookstore.online.dto.category.CategoryRequestDto;
 import ua.bookstore.online.dto.category.CategoryResponseDto;
+import ua.bookstore.online.dto.order.OrderItemResponseDto;
+import ua.bookstore.online.dto.order.OrderResponseDto;
+import ua.bookstore.online.dto.shopping.cart.CartItemRequestDto;
+import ua.bookstore.online.dto.shopping.cart.CartItemResponseDto;
+import ua.bookstore.online.dto.shopping.cart.QuantityDto;
+import ua.bookstore.online.dto.shopping.cart.ShoppingCartDto;
+import ua.bookstore.online.dto.user.UserRegistrationRequestDto;
+import ua.bookstore.online.dto.user.UserResponseDto;
 import ua.bookstore.online.model.Book;
+import ua.bookstore.online.model.CartItem;
 import ua.bookstore.online.model.Category;
-
+import ua.bookstore.online.model.Order;
+import ua.bookstore.online.model.OrderItem;
+import ua.bookstore.online.model.Role;
+import ua.bookstore.online.model.RoleName;
+import ua.bookstore.online.model.ShoppingCart;
+import ua.bookstore.online.model.User;
 
 public class ConstantAndMethod {
```

Comment:
I think it is safe to say that this class mostly provides some test data, so
```suggestion
public class DataUtils {
```
and let's add private constructor for it, and maybe even throw some exception there, on attempt to call it